src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -97,6 +106,17 @@ public void handleObservers(JButton button) {
         this.isDeleted = !this.isDeleted;
     }
 
+    private void loadCommandsFromJSON() {
+        String jsonInput = InputCommandsTextArea.getText().trim();
+        JSONCommandParser jsonCommandParser = new JSONCommandParser(jsonInput);
+        jsonCommandParser.parseToDriverCommand();
```

Comment:
jsonInput should be a parameter of this method to re-use parser. A jsonCommandParser instance could be then re-used (stored in the CommandManagerWindow instance as an attribute)